tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -315,6 +320,141 @@ function ( Request $request ) use ( $property_id, $webdatastream_id, $measuremen
 		);
 	}
 
+	public function data_create_account_ticket_required_parameters() {
+		return array(
+			'displayName'    => array( 'displayName' ),
+			'regionCode'     => array( 'regionCode' ),
+			'propertyName'   => array( 'propertyName' ),
+			'dataStreamName' => array( 'dataStreamName' ),
+			'timezone'       => array( 'timezone' ),
+		);
+	}
+
+	/**
+	 * @dataProvider data_create_account_ticket_required_parameters
+	 */
+	public function test_create_account_ticket__required_parameters( $required_param ) {
+		$valid_params = array(
+			'displayName'    => 'test account name',
+			'regionCode'     => 'US',
+			'propertyName'   => 'test property name',
+			'dataStreamName' => 'test stream name',
+			'timezone'       => 'UTC',
+		);
+
+		$provision_account_ticket_request = null;
+		FakeHttp::fake_google_http_handler(
+			$this->analytics->get_client(),
+			function ( Request $request ) use ( &$provision_account_ticket_request ) {
+				$url = parse_url( $request->getUri() );
+
+				if (
+					'sitekit.withgoogle.com' === $url['host']
+					&& '/v1beta/accounts:provisionAccountTicket' === $url['path']
+				) {
+					$provision_account_ticket_request = $request;
+				}
+
+				return new Response( 200 );
+			}
+		);
+
+		$this->analytics->register();
+		// Grant required scopes.
+		$this->authentication->get_oauth_client()->set_granted_scopes(
+			array_merge(
+				$this->authentication->get_oauth_client()->get_required_scopes(),
+				(array) Analytics::EDIT_SCOPE
+			)
+		);
+
+		$data = $valid_params;
```

Comment:
It's another minor niggle, but this assignment to `$data` is unnecessary, why not just refer to `$valid_params` throughout?